docs/querying/api.md

```diff
@@ -683,6 +683,46 @@ $ curl http://localhost:9090/api/v1/status/flags
 
 *New in v2.2*
 
+###  Runtime Info
+
+The following endpoint returns runtime info of the Prometheus server.
+
+*Warning: The contents of this api may change over time.*
```

Comment:
I'd weaker this a bit, maybe "Note that the exact values returned may change over time".